content/operate/kubernetes/deployment/openshift/openshift-operatorhub.md

```diff
@@ -79,6 +79,8 @@ The **Installed Operators**->**Operator details** page shows the provided APIs:
 
 Use the YAML view to create a custom resource file or let OperatorHub generate the YAML file for you by specifying your configuration options in the form view.
 
+<note> The REC name cannot be changed after cluster creation.</note>
```

Comment:
Does this need to go also into the OpenShift CLI install doc? I'm not sure this is _that_ significant of a remark to be repeated everyone, but since this is already repeated between the general install and the OLM one...